apps/cowswap-frontend/src/modules/permit/types.ts

```diff
@@ -0,0 +1,62 @@
+import { latest } from '@cowprotocol/app-data'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { Web3Provider } from '@ethersproject/providers'
+import { Token } from '@uniswap/sdk-core'
+
+import { Eip2612PermitUtils } from '@1inch/permit-signed-approvals-utils'
+
+export type PermitType = 'dai-like' | 'eip-2612'
```

Comment:
ohh, i see it now here the change. Not sure how before i saw the old version. Looks great